package.json

```diff
@@ -9,7 +9,8 @@
     "eslint-plugin-json": "1.2.0",
     "lintspaces-cli": "0.6.0",
     "stylelint": "7.9.0",
-    "webpack": "2.6.1"
+    "webpack": "2.6.1",
+    "private": true
```

Comment:
Now this is in the wrong place again (you were nearly correct just now). Note that my comment in the original issue is a patch - it shows a line to delete and a line to add.